Mathlib/Data/Finsupp/Defs.lean

```diff
@@ -661,6 +661,14 @@ theorem erase_ne {a a' : α} {f : α →₀ M} (h : a' ≠ a) : (f.erase a) a' =
   classical simp only [erase, coe_mk, h, ite_false]
 #align finsupp.erase_ne Finsupp.erase_ne
 
+theorem erase_apply [DecidableEq α] {a a' : α} {f : α →₀ M} :
```

Comment:
Arguably these should all be explicit

Reply:
I tend to agree, but I also see that semantically similar lemmas nearby leave them implicit. I think I'll leave this for a future PR to correct, hope that's ok.